src/utils/search/topic-extractor.ts

```diff
@@ -2,8 +2,8 @@ import {QueryReturnType} from '@/lib/search-url-builder'
 import {first, isEmpty} from 'lodash'
 
 export const topicExtractor = (searchState: QueryReturnType) => {
-  if (!isEmpty(searchState.refinementList?._tags)) {
-    return searchState.refinementList?._tags
+  if (!isEmpty(searchState?.refinementList?._tags)) {
+    return searchState?.refinementList?._tags
```

Comment:
sometimes we don't have `searchState` if we are returning an error so this does a safe look up